app/models/group.rb

```diff
@@ -521,9 +521,20 @@ def usernames
 
   PUBLISH_CATEGORIES_LIMIT = 10
 
-  def add(user)
+  def add(user, notify: false)
     self.users.push(user) unless self.users.include?(user)
 
+    if notify
+      Notification.create!(
+        notification_type: Notification.types[:group_invite],
```

Comment:
I would actually call this `:membership_request_accepted` ... `group_invite` is a bit confusing as a name for this thing.